views/employees.py

```diff
@@ -102,10 +102,14 @@ def employee_uri():
             if empl_id:
                 for x in empl_id:
```

Comment:
The previous comment relates to the variable name for `empl_id`, so you don't have to worry about it.